packages/core/src/components/menu/menu.md

```diff
@@ -83,25 +83,34 @@ To add a submenu to a `Menu`, simply nest `MenuItem`s within another `MenuItem`.
 The submenu opens to the right of its parent by default, but will adjust and flip to the left if
 there is not enough room to the right.
 
+`Menu` provides two `submenu` props to adjust this flipping behavior: you can customize the boundary element
+and the padding within that boundary element. Note that a `MenuItem` _must be inside_ a `Menu` element
+for these props to affect the submenus. On standalone `MenuItem`s (without a parent `Menu`, an anti-pattern) you can
+use `popoverProps.modifiers` directly to achieve the same result.
```

Comment:
If it's an anti-pattern, why even mention it?

Reply:
yeah i had this exact thought. i figure it's going to happen (is already happening) so why not pass judgment on it.